include/boost/json/parser.hpp

```diff
@@ -29,16 +29,17 @@ BOOST_JSON_NS_BEGIN
 
     @par Usage
 
-    Before parsing a new JSON, the function @ref reset
-    must be called, optionally passing the storage
-    pointer to be used by the @ref value container into
-    which the parsed results are stored. After the
-    parse is started, call the @ref write function
-    to provide buffers of characters of the JSON.
-    When there are no more buffers, call @ref finish.
-    The parse is complete when the function @ref done
-    returns `true`, or when a non-successful error
-    code is returned.
+    Parsing for a new JSON may begin the parser is
```

Comment:
This sentence looks broken... possibly meant to say 'when the parser is'

Reply:
This will be fixed elsewhere

Reply:
Ack